Source/WebKit/Shared/Cocoa/ArgumentCodersCocoa.mm

```diff
@@ -96,16 +96,16 @@ - (id)archiver:(NSKeyedArchiver *)archiver willEncodeObject:(id)object
         return adoptNS([[WKSecureCodingURLWrapper alloc] initWithURL:unwrappedURL]).autorelease();
 
     if (auto mutableArray = dynamic_objc_cast<NSMutableArray>(object); mutableArray && rewriteMutableArray)
-        return [mutableArray copy];
+        return adoptNS([mutableArray copy]).autorelease();
```

Comment:
Why is this better than just `[[mutableArray copy] autorelease]` ?

Reply:
It's not /really/, but it may make the ARC transition more incremental, I think, is the argument.